### PR TITLE
IP-mate-academy / jv-logger

### PR DESCRIPTION
**Pull request by IParov**

### CODE REVIEW

File: src/main/java/mate/academy/service/AuthenticationServiceImpl.java [hunk 1]
```diff
@@ -2,11 +2,15 @@
...
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
-        //TODO: add corresponding log message about method login was called
+        logger.info("method login was called. Params: {}", login);
```
Comment (Rommelua) [inline src/main/java/mate/academy/service/AuthenticationServiceImpl.java:13]:
```suggestion
        logger.info("Method login was called. Params: login = {}", login);
```

File: src/main/java/mate/academy/service/OrderServiceImpl.java [hunk 2]
```diff
@@ -4,11 +4,15 @@
...
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
+        logger.info("method completeOrder was called. Params: " + userId);
```
Comment (Rommelua) [inline src/main/java/mate/academy/service/OrderServiceImpl.java:15]:
```suggestion
        logger.info("Method completeOrder was called. Param: userId = {}", userId);
```

File: src/main/java/mate/academy/service/OrderServiceImpl.java [hunk 3]
```diff
@@ -24,7 +28,7 @@ private List<Product> getAllProductsFromShoppingCart(Long userId) {
         Product macBook = new Product("MacBook Air 2020", BigDecimal.valueOf(1399));
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
-        // TODO: add log message about successful fetched data from DB
+        logger.info("data successfully received from DataBase");
```
Comment (Rommelua) [inline src/main/java/mate/academy/service/OrderServiceImpl.java:31]: logger.info("Successful fetched data from DB. "
                + "Params: userId = {}, products = {}", userId, products);

File: .gitignore [hunk 4]
```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+logs/
```
Comment (Rommelua) [inline .gitignore:4]:
```suggestion
logs/*
```

**Review by Rommelua — CHANGES_REQUESTED** [review]

**Review by andriy-shulepa — APPROVED** [review]

**Pull request closed without merging**